src/søknad/steg/4-barnasbosted/OmAndreForelder.tsx

```diff
@@ -138,16 +131,23 @@ const OmAndreForelder: React.FC<Props> = ({
       },
     };
 
+    let verdi = '';
```

Comment:
Generelt bedre med const. ALternativt en funksjon `utledIkkeOppgittAnnenForelderBegrunnelse()`

Reply:
hvis du bytter plass på disse to kan også nyForelder være `const`

Reply:
Endret til å bruke const.